### ISSUE TITLE
Passing Token as parameter in request return error code 400:token_not_present

### ISSUE THREAD

**Issue by ben-x** [tymondesigns/jwt-auth#947]
(Jwt-auth version 0.5.9)
Tried sending the token as part of my api request from angularjs but gets a status code of 400, token_not_present.
After much investigation, I found out the issue was coming from the parseToken method of  JWTAuth.php
The $request->query($query, false) method always return false, for some reasons am yet to find out.
I tried modifying that line from $request->query($query, false) to $request->input('token') and then everything seem to work out fine.
However, sending the token as part of the header using "Authorization Bearer {token}" works fine.
In conclusion, I think the part that parse the sent as request data has been overlook or not properly tested and has been so since most developer choose to attach the token to the headers.

I hope this observation saves someone from hours of figuring out why his/her code blocks isn't working.

**Comment by tymondesigns** [comment]
Closing old issues. If still an issue please re-open
